### DISCUSSION TITLE
When using the ngrinder cluster, the following errors occur. What is the reason.

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#844] [Q&A]
![image](https://user-images.githubusercontent.com/50607317/141711214-bd24fa3d-e44c-4812-b857-31c8fe541e2e.png)

Linxu server has no Internet，
@junoyoon

**Comment by junoyoon** [comment]
shutdown all server first. and assure that all server's version is same.

**Reply by zhougy1989** [reply]
the server is same, the version is v3.4.4

**Reply by zhougy1989** [reply]
linux server is centos 7,Does this have anything to do with the Internet？

**Reply by junoyoon** [reply]
We don't support 3.4.X anymore. Please upgrade to 3.5.X

By the way. I highly suggest not to use nGrinder in the no internet connection env.

**Reply by zhougy1989** [reply]
thks,i will try it

**Reply by zhougy1989** [reply]
![image](https://user-images.githubusercontent.com/50607317/141726421-642a9514-33b0-4570-9253-eb2b6566c7a4.png)
when i user v3.5.5,start the server ,the log tell me H2 lost a table,

**Reply by junoyoon** [reply]
@imbyungjun  check it

**Reply by imbyungjun** [reply]
The error seems like related to https://github.com/h2database/h2database/issues/1067. 
> Support for synonyms that was merged in 1.4.196 broke compatibility with previous versions. This issue happens only if newer client is used with an older server.

nGrinder uses embedded h2 version **1.4.197**. If you running the H2 server version below 1.4.196, you should update your H2 server.
